### PR TITLE
Add missing dependencies

### PR DESCRIPTION
**Pull request by ekropotin**
Hello! I've just started test project implementation and noticed that there are some dependencies   missed in package.json

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -18,6 +18,7 @@
     "@types/react-dom": "16.0.3",
     "@types/webpack": "3.8.1",
     "@types/webpack-env": "1.13.2",
+    "@types/react-redux": "^5.0.14",
```
Comment (ivstas) [inline package.json]: Despite this repository was created using`typescript-react-redux-boilerplate` boilerplate, there is no `redux` in dependencies. As it was told, you can use any state library of your choice, so there is no need to add this type definitions.

**Comment by Corey-Maler** [comment]
Thank you for your pull request!

As @ivstas said before, our task can be done with every state management library which do you want, we don't want to set limits. That's why better way is removing file `types/react-redux` (and probably unused file `types/models`) instead adding `@types/react-redux` in `package.json`. Can you please fix this?

Fontawesome dependencies were last by mistake, so thank you for fixing it.

**Comment by ekropotin** [comment]
Fixed your comments. Please take a look.

**Comment by Corey-Maler** [comment]
Now everything is nice, thank you!

**Pull request merged**